src/julia/Integer.jl

```diff
@@ -58,10 +58,14 @@ function expressify(a::Integer; context = nothing)
     return a
 end
 
-function show(io::IO, R::Integers)
+function show(io::IO, R::Integers{BigInt})
    print(io, "Integers")
 end
 
+function show(io::IO, R::Integers{T}) where T
+   print(io, "Integers{$T}()")
```

Comment:
This is of course not needed for this PR but I found it very confusing that all these different rings print identically. But I can also drop it here, or change it (perhaps @lgoettgens or @thofma or @fieker have better ideas? Or anyone else?)

Reply:
I wish they would not exist.

Reply:
@thofma I'll read that as "I don't care, just don't mention them while I am closeby ;-) "